internal/core/application/service.go

```diff
@@ -1032,7 +1037,37 @@ func (s *Service) IncreaseInboundCapacity(ctx context.Context, amount uint64) (s
 		return "", fmt.Errorf("invalid preimage: %v", err)
 	}
 
-	return s.ClaimVHTLC(ctx, decodedPreimage)
+	txid, err := s.ClaimVHTLC(ctx, decodedPreimage)
+
+	// Create Swap Data To Store
+	swapData := domain.Swap{
+		Id:         swap.Id,
+		Amount:     amount,
+		Date:       time.Now(),
+		Status:     domain.SwapPending,
+		Invoice:    swap.Invoice,
+		RedeemTxId: txid,
+		To:         boltz.CurrencyArk,
+		From:       boltz.CurrencyBtc,
+	}
+
+	if err != nil {
+		swapData.Status = domain.SwapFailed
+		err = s.dbSvc.Swap().Add(ctx, swapData)
+		if err != nil {
+			return "", fmt.Errorf("failed to store swap data: %v", err)
+		}
+		return "", fmt.Errorf("failed to claim vHTLC: %v", err)
+	}
+
+	swapData.Status = domain.SwapSuccess
+	err = s.dbSvc.Swap().Add(ctx, swapData)
+	if err != nil {
+		return "", fmt.Errorf("failed to store swap data: %v", err)
+	}
```

Comment:
move this code into a go routine so it doesn't affect response time:

```suggestion
go func() {
	// Create Swap Data To Store
	swapData := domain.Swap{
		Id:         swap.Id,
		Amount:     amount,
		Date:       time.Now(),
		Status:     domain.SwapPending,
		Invoice:    swap.Invoice,
		RedeemTxId: txid,
		To:         boltz.CurrencyArk,
		From:       boltz.CurrencyBtc,
	}

	if err != nil {
		swapData.Status = domain.SwapFailed
		err = s.dbSvc.Swap().Add(ctx, swapData)
		if err != nil {
			return "", fmt.Errorf("failed to store swap data: %v", err)
		}
		return "", fmt.Errorf("failed to claim vHTLC: %v", err)
	}

	swapData.Status = domain.SwapSuccess
	err = s.dbSvc.Swap().Add(ctx, swapData)
	if err != nil {
		return "", fmt.Errorf("failed to store swap data: %v", err)
	}
}()	
```